.changeset/dirty-seahorses-look.md

```diff
@@ -0,0 +1,5 @@
+---
+"geohub": minor
+---
+
+feat: registering of algorithms
```

Comment:
please merge two changeset files into a file. a PR should only have a changeset file for each package/site.